src/resources/closure.lib.d.ts

```diff
@@ -5,3 +5,7 @@ declare namespace ಠ_ಠ.clutz_internal {
   type GlobalError = Error;
   var GlobalError: ErrorConstructor;
 }
+
+// TODO: does this match ES6 closure semantics? Check if all
+// goog fields accessible in ES6 modules.
```

Comment:
As discussed offline, please document that all the symbols in the `goog` namespace are magically added by the Closure-specific compiler pass.